internal/context/context.go

```diff
@@ -1,8 +1,9 @@
 package context
 
 type CLIContext struct {
-	Config CLIConfig
-	Paths  Paths
+	Config              CLIConfig
+	Paths               Paths
+	ProjectConfigExists bool
```

Comment:
This is an addition in order to mark (on discovery) if the project config yaml exists.